src/interfaces/general.ts

```diff
@@ -4,10 +4,26 @@ export type GenericObjectInterface<T> = {
 };
 
 export type StringObject = GenericObjectInterface<string>;
+export type NumberObject = GenericObjectInterface<number>;
+
+export type Nullable<T> = T | null;
 
 export interface FontMakerOptions {
   size?: number;
   color?: string;
   weight?: string;
   family?: string;
 }
+
+declare global {
+  interface Console {
+    tron: any;
+  }
+}
+
+export interface ReactObjectType {
```

Comment:
i think this name is not clear, what u think? maybe something related with hitSlop?